packages/components/pages/f-mfa/stories/story.helper.js

```diff
@@ -0,0 +1,98 @@
+import {
+    SUBMIT_LOGIN_CHALLENGE_URL
+} from '../src/constants';
+
+const httpStatusCodes = {
+    ok: 200,
+    badrequest: 400,
+    throttledrequest: 429
+};
+
+const httpVerbs = {
+    post: 'POST'
```

Comment:
could just pass this in directly without the const.

Reply:
By creating then as an enumerate of constant values you can use them multiple times within the definition section and never be concerned about typo bugs.